bootstrap/config/kfctl_gcp_basic_auth.yaml

```diff
@@ -288,64 +279,43 @@ spec:
         name: manifests
         path: gcp/gpu-driver
     name: gpu-driver
-  - kustomizeConfig:
-      overlays:
-      - application
-      repoRef:
-        name: manifests
-        path: seldon/seldon-core-operator
-    name: seldon-core-operator
-  - kustomizeConfig:
-      parameters:
-      - name: ambassadorServiceType
-        value: NodePort
-      - name: namespace
-        value: istio-system
-      repoRef:
-        name: manifests
-        path: common/ambassador
-    name: ambassador
   - kustomizeConfig:
       repoRef:
         name: manifests
         path: common/basic-auth
     name: basic-auth
   - kustomizeConfig:
-      overlays:
-      - certmanager
-      - gcp-credentials
       parameters:
       - name: namespace
         value: istio-system
       - initRequired: true
         name: ipName
-        value: test1-ip
+        value: kf101-ip
       - initRequired: true
         name: hostname
-        value: test1.endpoints.SET_PROJECT.cloud.goog
+        value: kf101.endpoints.SET_PROJECT.cloud.goog
       - initRequired: true
         name: project
         value: SET_PROJECT
-      - name: ingressName
-        value: envoy-ingress
-      - name: issuer
-        value: letsencrypt-prod
       repoRef:
         name: manifests
         path: gcp/basic-auth-ingress
     name: basic-auth-ingress
   email: SET_EMAIL
   enableApplications: true
+  hostname: kf101.endpoints.SET_PROJECT.cloud.goog
+  ipName: kf101-ip
   packageManager: kustomize
   platform: gcp
+  project: SET_PROJECT
+  repos:
+  - name: kubeflow
```

Comment:
can you keep the comment:
 # To get manifest at a PR:
   #uri: https://github.com/kubeflow/manifests/archive/pull/235/head.tar.gz

I think it's useful